logic/src/commonMain/kotlin/com/wire/kalium/logic/data/auth/login/SSOLoginRepository.kt

```diff
@@ -21,6 +22,8 @@ interface SSOLoginRepository {
 
     suspend fun finalize(cookie: String, serverConfig: ServerConfig): Either<NetworkFailure, String>
 
+    suspend fun ssoEstablishSession(cookie: String, serverConfig: ServerConfig): Either<NetworkFailure, SessionDTO>
```

Comment:
Do we need the `sso` prefix? I think that if we're using `SSOLoginRepository` that's saying enough that it's being used for sso, at least for me 😄